barcode/writer.py

```diff
@@ -191,14 +192,19 @@ def render(self, code):
                     c += 1
                 else:
                     if line[i] == "1":
-                        mlist.append(c)
+                        mlist.append((c, 1))
+                    elif line[i] == "G":
+                        mlist.append((c, 1.1))
                     else:
-                        mlist.append(-c)
+                        mlist.append((-c, 1))
```

Comment:
Looks like images are rendered with the guards slightly "fatter" in their lower end.

![image](https://user-images.githubusercontent.com/730811/103030281-668c3980-4553-11eb-98e5-dbe6029cf79b.png)

Making the white lines the same length as the G-lines seems to fix it.

```suggestion
                        mlist.append((-c, 1.1))
```